scripts/compose-dev.yml

```diff
@@ -10,7 +10,7 @@ services:
     environment:
       - TMUX
       - TEST
-      - GALAXY_DB_URL=postgres://postgres:postgres@postgres:5432/galaxy
+      - GALAXY_DB_URL=postgres://galaxy:galaxy@postgres:5432/galaxy
```

Comment:
Connect as `galaxy` user, and not `postgres`. Connecting as `postgres` causes new database objects to be owned by `postgres`.